### ISSUE TITLE
Width of company cards

### ISSUE THREAD

**Issue by JonRojas** [JonRojas/dc_jobs#5]
Would like to see the default card width equal to the card with the greatest width

**Comment by timfoley** [comment]
Addressed this with the "fixed width issue" pull-request (#8). 
Here's a tidbit of the code. 

```
function resizeCard() {
  $('.profile-card + .profile-card').css({
    'max-width': $('.profile-card:first-child').width()
    });
}
```

**Comment by JonRojas** [comment]
Closing Issue.
